tests/unit/jest.config.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const rootDir = '../../';
+const currentDir = '<rootDir>/tests/unit';
+
+module.exports = {
+    clearMocks: true,
+    testRunner: 'jest-circus/runner',
+    transform: {
+        '^.+\\.(ts|tsx)$': 'ts-jest',
```

Comment:
I don't this this project should have tsx files; does our jest config need to support them? (same for lines 15, 20, 26)